### PR TITLE
Set up linting via Vale

### PR DESCRIPTION
**Pull request by juliasilge**
This PR adds the [Vale linter](https://github.com/rstudio/docs.rstudio.com/wiki/Vale-Linter-%E2%80%90-Style-Guide-Compliance) for our Positron docs, to help automate some of our style guide compliance.

### CODE REVIEW

**Comment by juliasilge** [comment]
I'm going to troubleshoot the GH action separately, just pushing to main, because I can't get it to fail here on the PR. 🤔

File: data-explorer.qmd [hunk 1]
```diff
@@ -31,7 +31,7 @@ To open a new instance of the Data Explorer on a specific data frame, use one of
 
 ## Data grid
 
-The data grid is the primary display, with a spreadsheet-like cell by cell view. it's intended to scale efficiently to relatively large in-memory datasets, up to millions of rows or columns. Each column header has the column name above the data type as used in the language runtime. There is a context menu at the top right of each column to control sorting or quick-add a filter for the selected column. Columns are resizable by click and drag of the column borders. 
+The data grid is the primary display, with a spreadsheet-like cell by cell view. It's intended to scale efficiently to relatively large in-memory datasets, up to millions of rows or columns. Each column header has the column name above the data type as used in the language runtime. There is a context menu at the top right of each column to control sorting or quick-add a filter for the selected column. Columns are resizable by click and drag of the column borders.
```
Comment (AshleyHenry15) [inline data-explorer.qmd:34]: I am not certain if the data grid is the same as the RStudio IDE Data Viewer. If it is, we should be consistent with our naming: https://docs.posit.co/ide/user/ide/guide/data/data-viewer.html

```suggestion
The data grid is the primary display, with a spreadsheet-like cell-by-cell view. It's intended to scale efficiently to relatively large in-memory datasets, up to millions of rows or columns. Each column header has the column name above the data type, as used in the language runtime. At the top right of each column, there is a context menu to control sorting or to add a filter for the selected column. Resize columns by clicking and dragging the column's borders. 
```

File: start.qmd [hunk 2]
```diff
@@ -38,7 +38,7 @@ Positron communicates with Python via the IPykernel package, so it needs to be i
 python3 -m pip install ipykernel
 ```
 
-Alternatively, Positron will prompt you to install the IPykernel package if it'sn't found when you first try to use a Python environment.
+Alternatively, Positron will prompt you to install the IPykernel package if it isn't found when you first try to use a Python environment.
```
Comment (AshleyHenry15) [inline start.qmd:41]:
```suggestion
Alternatively, Positron prompts you to install the IPykernel package if it isn't found when you initially try to use a Python environment.
```

**Review by AshleyHenry15 — APPROVED** [review]
Minor suggestions.

File: data-explorer.qmd (hunk 1 above)
Comment (juliasilge) [reply data-explorer.qmd:34]: The feature in Positron is called "Data Explorer" to differentiate it from RStudio's Data Viewer, and here we are using lowercase "data grid" to refer to the waffle/grid section of the feature, as opposed to the areas with summary information. This is a new feature (not in VS Code) that we will need to document pretty heavily, and we would definitely be up for iterating on our initial attempt here in partnership with you!

**Pull request merged**